CHANGES/8455.feature.rst

```diff
@@ -0,0 +1 @@
+Added `ClientConnectorDNSError` for differentiating DNS resolution errors from other connector errors.
```

Comment:
Plz use RST, not MD.

Reply:
Here's a similar example: https://github.com/aio-libs/aiohttp/pull/6722/files#diff-06a2c33bab93f9984fab90f699c24c12c18cd0620eec8555deef04ae43b92925